File: Tareas/Unidad2/a2253332351_practica01/03_AreaCono.java
import java.util.Scanner;

public class AreaCono {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        double radio, generatriz, altura;
        double aBase, aLado, aTotal, volumen;
        final double PI = Math.PI;

            System.out.println("Ingresa el radio del cono: ");
            radio = entrada.nextDouble();
            System.out.println("Ingresa la generatriz del cono: ");
            generatriz = entrada.nextDouble();
            System.out.println("Ingresa la altura del cono: ");
            altura = entrada.nextDouble();

            aBase = PI * (radio * radio);
            aLado = PI * radio * generatriz;
            aTotal = aBase + aLado;
            volumen = (aBase * altura) / 3;

            System.out.println("El área de la base es: " + aBase);
            System.out.println("El área lateral es: " + aLado);
            System.out.println("El área total es: " + aTotal);
            System.out.println("El volumen es: " + volumen);
     }
}

File: Tareas/Unidad2/a2253332351_practica01/03_AreaCono.py
# AreaCono.py

radio = float(input("Ingresa el radio del cono: "))
generatriz = float(input("Ingresa la generatriz del cono: "))
altura = float(input("Ingresa la altura del cono: "))

PI = 3.14159265358979323846

aBase = PI * (radio * radio)
aLado = PI * radio * generatriz
aTotal = aBase + aLado
volumen = (aBase * altura) / 3

print(f"El área de la base es: {aBase}")
print(f"El área lateral es: {aLado}")
print(f"El área total es: {aTotal}")
print(f"El volumen es: {volumen}")
